src/current/cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -195,7 +199,11 @@ To configure a custom SAML authentication method using the service provider-init
 1. Optionally, [configure advanced settings](#configure-advanced-settings) for the new authentication method.
 1. Download metadata required by your IdP. Click **Download**. Open the file and make a note of the following values:<ul><li><b>Entity ID</b>: The <code>entityID</code> attribute of the <code>&lt;EntityDescriptor&gt;</code> tag.</li><li><b>Login URL</b>: The <code>location</code> attribute of the <code>&lt;AssertionConsumerService&gt;</code> tag.</li></ul>
 1. In the browser where you are logged in to your IdP, update the authentication configuration to use the Entity ID and Login URL from the metadata file.
-1. Configure the SAML assertions that your IdP sends to CockroachDB {{ site.data.products.cloud }}. Your IdP must send an assertion with a `name` field and a second assertion with an `email` field, each mapped to the relevant fields in your IdP. Refer to the documentation for your IdP.
+1. Configure the SAML assertions that your IdP sends to CockroachDB {{ site.data.products.cloud }}.
+
+    Your IdP must send an assertion with a `name` field and a second assertion with an `email` field, each mapped to the relevant fields in your IdP. Otherwise, [autoprovisioning](#autoprovisioning) for the authentication method will fail, and users will need to be invited before they can sign in. To configure the SAML assertion, refer to the documentation for your IdP.
```

Comment:
This isn't related to autoprovisioning - all logins will fail if these fields are not set.

Reply:
Oh, I thought the problem was with needing to invite the user when they expected them to be autoprovisioned. I'll update.